src/features/page_objects/legal_agreement/add-legal-agreement-parties

```diff
@@ -0,0 +1,26 @@
+const Page = require("../page");
+
+class LegalAgreementAddParties extends Page {
+  get titleText() {
+    return "Add all parties listed on the legal agreement";
+  }
+
+  get legalPartyName() {
+    //Todo : return $("#organisation[0][organisationName]");
+    return $(".govuk-input");
+  }
+    
+  get legalPartyRole(){
+    //Todo: fix when html on page is fixed and all ids are unique
+    return $(".govuk-radios__input");
+  }
+
+  get legalContinueButton() {
+    return $("#continue");
```

Comment:
Do you have different selectors for continue/submit form buttons on each page or could you just have one in the base page?

Reply:
Ive had to ask the dev to add a generic id for it as currently its been added seperately